### PR TITLE
[Keymap] Add keymap for keebio/quefrency 65% version

### PR DESCRIPTION
**Pull request by GeorgePetri**
## Description

Add a new keymap. 

## Types of Changes
- [x] Keymap/layout/userspace (addition or update)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/keebio/quefrency/keymaps/GeorgePetri/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# George Petri's Quefrency 65 layout
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/keymaps/GeorgePetri/README.md]: Could you rename this so the filename is fully lowercase?

e.g. `keyboards/keebio/quefrency/keymaps/GeorgePetri/readme.md`

File: keyboards/keebio/quefrency/keymaps/GeorgePetri/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+```
+make keebio/quefrency:GeorgePetri
+```
+
+Based on the default querty layout with minor tweaks. 
+The position of the arrow keys in a line in the bottom right. 
+The backspace key is 1u and to the left of the delete key.
+Grave, pause and insert are on the fuction layer.
```
Comment (noroadsleft) [inline keyboards/keebio/quefrency/keymaps/GeorgePetri/README.md]: Spelling fix.

```suggestion
Grave, pause and insert are on the function layer.
```

**Comment by GeorgePetri** [comment]
I've renamed, and fixed the typo.

File: keyboards/keebio/quefrency/keymaps/GeorgePetri/readme.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
+# George Petri's Quefrency 65 layout
+
+```
+make keebio/quefrency:GeorgePetri
+```
+
+Based on the default querty layout with minor tweaks.
```
Comment (drashna) [inline keyboards/keebio/quefrency/keymaps/GeorgePetri/readme.md]: You need to have double spaces at the end of each line, or they won't properly break.   If you view the file on github, you'll see what I mean.

**Comment by drashna** [comment]
Well, could you rename the keymap folder, specifically?  

If you did, .... that's part of why we want all lower case: it didn't "take".   You may need to rename to something different (eg "temp"), and then rename back to "georgepetri".

**Comment by GeorgePetri** [comment]
I've renamed the directory.

**Comment by drashna** [comment]
Awesome, thank you!

**Pull request merged**